src/engine/engine.go

```diff
@@ -321,7 +332,7 @@ func (e *Engine) PrintTooltip(tip string) string {
 		Segments:  []*Segment{tooltip},
 	}
 	switch e.Env.Shell() {
-	case shell.ZSH, shell.CMD, shell.FISH:
+	case shell.ZSH, shell.CMD, shell.FISH, shell.PLAIN:
```

Comment:
why was this added? It's not really used anywhere as it's not a valid shell name.

Reply:
So this because if the `Shell()` call returns an empty string, the case statement is skipped and thus an empty prompt string is returned as well.

Reply:
That could make sense, I'll add a sane default in that case.